installers/rpm-code/dnf-download.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set +euo pipefail
+source ~/.bashrc
+set -euo pipefail
+source ./dev-env.sh
+declare f
+sudo dnf makecache
+declare d=dnf-download-tmp
+rm -rf "$d"
+mkdir "$d"
+declare common=0
+for f in "$@"; do
+    dnf download --downloaddir "$d" "rscode-$f"
+    if [[ $1 == common ]]; then
+        common=1
+        cd ~/src/radiasoft/container-rpm-code
```

Comment:
Can you explain why the contents of this block (L16-22) would need to be run multiple times in this script? It's also duplicated below, so maybe should be made into a function

Reply:
2ac0988